### DISCUSSION TITLE
Need an organizer of files and desktop shortcuts, like in deepin.

### DISCUSSION THREAD

**Discussion by Rierkrol73jd8** [linuxmint/discussions#151] [Ideas]
several areas are created on the desktop.  When you add a shortcut to a program or file, these elements are moved to a specific area of the desktop.  areas should be scrollable.  well implemented in deepin, but it would be desirable if these areas could be moved manually.  and that these areas and widgets do not overlap each other.  At the moment, desktop shortcuts are covered by widgets.  translated as best I could😊

**Comment by anaximeno** [comment]
Can you show pictures or videos about that?

**Reply by Rierkrol73jd8** [reply]
I recorded a video from virtual box, because of this there are freezes. 
[deepin-screen0.webm](https://github.com/linuxmint/discussions/assets/144561132/a5249878-d4d9-4c21-a809-4a502f87e4af)

**Reply by anaximeno** [reply]
That's interesting. I've been thinking on something similar to that, but more close to how you can group applications in android (gnome applications section has something like that also), but it would work like in android when you click at it, it will expand to something like in the video you showed. In the case you show the organization is done automatically apparently, in what I suggest the user would have to drag and drop to create these groupings (or collections).

**Reply by Rierkrol73jd8** [reply]
It seems to me that it would be better if there was a manual and automatic organization of icons.  While studying you have to use a lot of materials and automatic sorting into groups helps you navigate faster.  ps.  translated using a software translator, I hope everything is translated clearly.

**Reply by rcalixte** [reply]
This seems like it would fit as an Extension (or maybe Desklet?) rather that needing to be featured into Nemo or the default desktop. This would likely be a sizeable endeavor for whoever would undertake it.

**Reply by anaximeno** [reply]
> This seems like it would fit as an Extension (or maybe Desklet?) rather that needing to be featured into Nemo or the default desktop. This would likely be a sizeable endeavor for whoever would undertake it.

I agree this behavior is more adequate as an extension in the context of Mint.
